static/app/components/replays/replayContext.tsx

```diff
@@ -130,7 +130,7 @@ const ReplayPlayerContext = React.createContext<ReplayPlayerContextProps>({
 
 type Props = {
   children: React.ReactNode;
-  replay: ReplayReader;
+  replay: ReplayReader | null;
```

Comment:
Note: this is only posslbe when the `static factory()` method can return `null`

See notes in #34571 34571